pygmt/helpers/utils.py

```diff
@@ -174,6 +174,41 @@ def _is_printable_ascii(argstr: str) -> bool:
     return all(32 <= ord(c) <= 126 for c in argstr)
 
 
+def _has_apostrophe_or_backtick(argstr: str) -> bool:
```

Comment:
"contains" sounds nicer than "has" (but it's longer).

```suggestion
def _contains_apostrophe_or_backtick(argstr: str) -> bool:
```